src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -349,5 +354,28 @@ void GcsPlacementGroupManager::OnNodeDead(const NodeID &node_id) {
   SchedulePendingPlacementGroups();
 }
 
+void GcsPlacementGroupManager::Tick() const {
+  UpdatePlacementGroupLoad();
+  execute_after(io_context_, [this] { Tick(); }, 1000 /* milliseconds */);
+}
+
+void GcsPlacementGroupManager::UpdatePlacementGroupLoad() const {
+  std::shared_ptr<rpc::PlacementGroupLoad> placement_group_load =
+      std::make_shared<rpc::PlacementGroupLoad>();
+  int total_cnt = 0;
+  RAY_LOG(ERROR) << "Sang, "
```

Comment:
Sang?

Reply:
Oh no.. lol that was for debugging 😂 I always write my name when doing print debugging lol